Source/Meadow.Foundation.Peripherals/Sensors.Environmental.NextPm/Driver/Sensors.Environmental.NextPm.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Meadow.Sdk/1.1.0">
+	<PropertyGroup>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageIcon>icon.png</PackageIcon>
+		<Authors>Wilderness Labs, Inc</Authors>
+		<TargetFramework>netstandard2.1</TargetFramework>
+		<OutputType>Library</OutputType>
+		<AssemblyName>NextPm</AssemblyName>
+		<Company>Wilderness Labs, Inc</Company>
+		<PackageProjectUrl>http://developer.wildernesslabs.co/Meadow/Meadow.Foundation/</PackageProjectUrl>
+		<PackageId>Meadow.Foundation.Sensors.Environmental.NextPm</PackageId>
+		<RepositoryUrl>https://github.com/WildernessLabs/Meadow.Foundation</RepositoryUrl>
+		<PackageTags>Meadow,Meadow.Foundation,Environmental,NextPM,tera,gas,particle,particulate,matter,AQI,air,quality</PackageTags>
+		<Version>0.1.0</Version>
+		<Nullable>enable</Nullable>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<Description>TERA Sensor NextPM particulate matter sensor</Description>
```

Comment:
Small thing we've standardized (but not documented), we include the communication type in the description so it shows up on the docs peripheral page

So "TERA Sensor NextPM serial particulate matter sensor"